llm_demo/evaluation/evaluation.py

```diff
@@ -0,0 +1,84 @@
+# Copyright 2024 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import json
+from typing import Any, Dict, List, Optional
+
+from pydantic import BaseModel, Field
+
+from orchestrator import BaseOrchestrator
+
+from .eval_golden import goldens
+
+
+class EvalData(BaseModel):
```

Comment:
It looks like we are using pydantic to convert from JSON to a Python object -- any reason to just not declare as a python object?

Reply:
Will update this in the eval-dataset PR.